readme.md

```diff
@@ -205,6 +213,30 @@ The [unique ID](#id-number) of the supported output device.
 
 The volume level to set the device to. Must be between 0 and 1, otherwise and error will be thrown.
 
+#### `getDeviceMute(deviceId: number, channelType: ChannelType? = null): Promise<void>`
+
+Get muting state for audio device.
+
+##### `deviceId: number`
+
+The [unique ID](#id-number) of the supported device.
+
+##### `channelType: ChannelType`
+
+[optional] [Channel type](#channeltype-enum) to get information from.
```

Comment:
Is this really optional? If we don't pass it we get an error right? Since in the swift code it defaults to null?

Reply:
Yes, it is optional. If some interface have only one channel we could omit this parameter.